sovtoken/sovtoken/main.py

```diff
@@ -35,7 +28,7 @@ def postTxnAddedToLedgerClbk(node, *args, **kwargs):
         node.ledger_ids.append(TOKEN_LEDGER_ID)
 
     node.ledgerManager.addLedger(TOKEN_LEDGER_ID, ledger,
-                                 postTxnAddedToLedgerClbk=functools.partial(postTxnAddedToLedgerClbk, node))
+                                 postTxnAddedToLedgerClbk=node.postTxnFromCatchupAddedToLedger)
```

Comment:
What does `Clbk` mean?

Reply:
this callback will be called for each txn from Catchup Reply. In this case we will do generic node's logic, like set to state and add to ledger. But for StaticFeesReqHandler i add another callback on ledger's caughtup complete for setting last committed into tracker.